.env.example_docker

```diff
@@ -31,6 +31,10 @@ KBIN_META_KEYWORDS="mbin, content aggregator, open source, fediverse"
 KBIN_HEADER_LOGO=false
 KBIN_FEDERATION_PAGE_ENABLED=true
 MBIN_DEFAULT_THEME=default
+MBIN_HOME=/var/www/mbin
```

Comment:
These en vars shouldn't be here as well ideally put these in the compose override yaml file.